apps/omg_eth/test/omg_eth/root_chain_test.exs

```diff
@@ -16,110 +16,112 @@ defmodule OMG.Eth.RootChainTest do
+        plasma_framework: contracts.plasma_framework
+      })
+
+    {:ok, _} = Support.DevHelper.transact_sync!(add_exit_queue)
+    on_exit(exit_fn)
     {:ok, contracts: contracts}
   end
 
   test "get_root_deployment_height/2 returns current block number", %{contracts: contracts} do
-    use_cassette "ganache/get_root_deployment_height", match_requests_on: [:request_body] do
-      {:ok, number} = RootChain.get_root_deployment_height(contracts.plasma_framework_tx_hash, contracts)
-      assert is_integer(number)
-    end
+    {:ok, number} = RootChain.get_root_deployment_height(contracts.plasma_framework_tx_hash, contracts)
+    assert is_integer(number)
   end
 
   test "get_next_child_block/1 returns next blknum to be mined by operator", %{contracts: contracts} do
-    use_cassette "ganache/get_next_child_block", match_requests_on: [:request_body] do
-      assert {:ok, 1000} = RootChain.get_next_child_block(contracts)
-    end
+    assert {:ok, 1000} = RootChain.get_next_child_block(contracts)
   end
 
   test "get_child_chain/2 returns the current block hash and timestamp", %{contracts: contracts} do
-    use_cassette "ganache/get_child_chain", match_requests_on: [:request_body] do
-      {:ok, {child_chain_hash, child_chain_time}} = RootChain.get_child_chain(0, contracts)
+    {:ok, {child_chain_hash, child_chain_time}} = RootChain.get_child_chain(0, contracts)
 
-      assert is_binary(child_chain_hash)
-      assert byte_size(child_chain_hash) == 32
-      assert is_integer(child_chain_time)
-    end
+    assert is_binary(child_chain_hash)
+    assert byte_size(child_chain_hash) == 32
+    assert is_integer(child_chain_time)
   end
 
   test "get_deposits/3 returns deposit events", %{contracts: contracts} do
-    use_cassette "ganache/get_deposits", match_requests_on: [:request_body] do
-      # not using OMG.ChildChain.Transaction to not depend on that in omg_eth tests
-      # payment tx_type, no inputs, one output, metadata
-      {:ok, deposit} = ExPlasma.Transaction.Deposit.new(owner: contracts.authority_address, currency: @eth, amount: 1)
-      rlp = ExPlasma.Transaction.encode(deposit)
-
-      {:ok, tx_hash} =
-        RootChainHelper.deposit(rlp, 1, contracts.authority_address, contracts)
-        |> DevHelper.transact_sync!()
-
-      {:ok, height} = Eth.get_ethereum_height()
-
-      authority_addr = contracts.authority_address
-      root_chain_txhash = Encoding.from_hex(tx_hash["transactionHash"])
-
-      deposits = RootChain.get_deposits(1, height, contracts)
-
-      assert {:ok,
-              [
-                %{
-                  amount: 1,
-                  blknum: 1,
-                  owner: ^authority_addr,
-                  currency: @eth,
-                  eth_height: height,
-                  log_index: 0,
-                  root_chain_txhash: ^root_chain_txhash
-                }
-              ]} = deposits
-    end
+    # not using OMG.ChildChain.Transaction to not depend on that in omg_eth tests
```

Comment:
not a matter of your change, but these 2 lines of comment are stale. Maybe let's take the opportunity and remove them